lib/transfer.js

```diff
@@ -156,18 +156,72 @@ function _changeDBFileField(file) {
     if (file.fileName == file.newFileName || !config.renameInDatabase) {
       return resolve();
     }
+
+    if (config.transferTo === 'filesystem'){
+      var _path = config.filesAdapter._filesDir + "/" + file.newFileName;
+    } else if (config.transferTo === 's3'){
+      var _path = "https://" + config.filesAdapter._bucket + "." + config.filesAdapter._s3Client.config.endpoint + "/" + config.filesAdapter._bucketPrefix + file.newFileName;
```

Comment:
there should be `getFileLocation` https://github.com/parse-server-modules/parse-server-s3-adapter/blob/master/index.js#L122 available from the S3 adapter, can we use that instead?